oioioi/contests/templates/contests/contest_rules.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base-with-menu.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+
+{% block title %}{% trans "Rules" %}{% endblock %}
+
+{% block main-content %}
+<h1>{% trans "Rules" %}</h1>
+<ul class="list-group list-group-light">
+    {% if scoring_type %}
+        {% blocktrans %}
```

Comment:
The recent syntax is `blocktranslate`. Does the text appear in the translation files?

Reply:
`blocktrans` and `trans` are still aliases for `blocktranslate` and `translate`